cmd/notation/list.go

```diff
@@ -43,16 +44,17 @@ func listCommand(opts *listOpts) *cobra.Command {
 			if opts.ociLayout {
 				opts.inputType = inputTypeOCILayout
 			}
-			return experimental.CheckFlagsAndWarn(cmd, "oci-layout")
+			return experimental.CheckFlagsAndWarn(cmd, "allow-referrers-api", "oci-layout")
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runList(cmd.Context(), opts)
 		},
 	}
 	opts.LoggingFlagOpts.ApplyFlags(cmd.Flags())
 	opts.SecureFlagOpts.ApplyFlags(cmd.Flags())
+	cmd.Flags().BoolVar(&opts.allowReferrersAPI, "allow-referrers-api", false, "[Experimental] use the Referrers API to list signatures, if not supported, fallback to the Referrers tag schema")
```

Comment:
Should we refactor it in `flags.go`?

Reply:
Since it's an experimental flag, I think we could keep it here? (similar to flag `--oci-layout`). There are `experimental.CheckFlagsAndWarn` and `experimental.HideFlags` in the file as well, maybe moving the flag into flags.go is not as clear as the current way?

Reply:
> Should we refactor it in `flags.go`?

Since it's an experimental flag, I think we could keep it here? (similar to flag `--oci-layout`). There are `experimental.CheckFlagsAndWarn` and `experimental.HideFlags` in the file as well, maybe moving the flag into flags.go is not as clear as the current way?

Reply:
> Should we refactor it in `flags.go`?

Refactored into flags.go